Dockerfile

```diff
@@ -12,8 +12,7 @@
 #
 
 # ----------- deps -----------
-# Install from Debian Stretch with modern Python support
-FROM python:slim-stretch as deps
+FROM python:3.7-stretch as deps
```

Comment:
This caught my eye. Are any services running Python 3.7 and using Microcosm? This version should be the lowest version of our Python services that use Microcosm, right?

Reply:
Yes, `sphinx` for [example](https://github.com/globality-corp/sphinx/blob/master/Dockerfile#L17). AFAIK none of them pin to specific patch version - I found this in our [Python wiki page](https://globality.atlassian.net/wiki/spaces/DEV/pages/927861472/Python#Standard/X86-Installation):

> We don’t pin to a specific bugfix release – you can install the latest.